core/committer/txvalidator/txvalidationflags.go

```diff
@@ -4,7 +4,7 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package util
+package txvalidator
```

Comment:
I think we should actually move this to `internal/pkg/txflags`.
`internal/pkg` was created for code that is shared across multiple packages.

Reply:
Nice. Thanks for the pointer. I wasn't aware of this `internal/pkg`. Will update the code.

Reply:
Yeah ... I'm glad you started on this work again.  I've been meaning to get back to (re)organizing a bunch of the code myself

Reply:
Yes, can't agree more. It is a much needed one.